src/components/Home/MainBenefitsCardsSection.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * MainBenefitsCardsSection is a component that represents
+ * the "Card" section of the Homepage page about a few things users may ask themself about using Avastar
+ */
+
+//utils
+import styled from 'styled-components';
+
+// Img
+import PadlockIcon from '../../assets/padlock.png';
+import EqualizerIcon from '../../assets/equalizer.png';
+import IdentityProfilIcon from '../../assets/identity-profil.png';
+
+import { Box, Typography } from '@mui/material';
+
+// Typescript types
+interface BenefitsCard {
+  title: string;
+  content: string;
+  icon: string;
+}
+
+// Contains each benefit's card informations
+const benefitsCard: BenefitsCard[] = [
+  {
+    title: 'Yes, your data is safe',
+    content: `Your data is safe with us as we can’t and don’t want to see your data. The files you upload never leave your computer, all your data is stored locally on your browser. It will also be deleted every time your session is refreshed.`,
+    icon: PadlockIcon,
+  },
+  {
+    title: 'No, you don’t need any technical skills',
+    content: `We remove the pain of manually analyzing the personal data you shared, so you can focus on managing your privacy. We provide you with all the information you need in order to understand the overview and act upon it.`,
```

Comment:
Juste remplacer le mot overview par data : "We provide you with all the information you need in order to understand **your data** and act upon it."

Reply:
Done.